WordPress/Classes/ViewRelated/Post/Prepublishing Nudge/PrepublishingHeaderView.swift

```diff
@@ -45,12 +46,21 @@ class PrepublishingHeaderView: UIView, NibLoadable {
     // MARK: - Title
 
     func setTitle(_ title: String?, transitionDuration: TimeInterval = 0.2) {
-        UIView.transition(with: publishingToLabel,
-                          duration: transitionDuration,
-                          options: .transitionCrossDissolve,
-                          animations: {
-            self.publishingToLabel.text = title?.uppercased()
-        })
+        titleLabel.text = title
+    }
+
+    func showTitle() {
+        titleLabel.layer.opacity = 1
+        blogTitleLabel.layer.opacity = 0
+        publishingToLabel.layer.opacity = 0
+        blogImageView.layer.opacity = 0
+    }
+
+    func hideTitle() {
+        titleLabel.layer.opacity = 0
+        blogTitleLabel.layer.opacity = 1
+        publishingToLabel.layer.opacity = 1
+        blogImageView.layer.opacity = 1
```

Comment:
Do we still need this since we're not displaying the navigation title anymore?

Reply:
Nope.

Removed in dc2bff296a124bf6ff60c0a9a595f4a2f7fdfc14